sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ContainerRegistryAudience.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Collection;
+
+/** Defines values for ContainerRegistryAudience. */
+public class ContainerRegistryAudience extends ExpandableStringEnum<ContainerRegistryAudience> {
+    /** Static value AzureResourceManagerChina for ContainerRegistryAudience. */
+    public static final ContainerRegistryAudience AZURERESOURCEMANAGERCHINA = fromString("https://management.chinacloudapi.cn");
+
+    /** Static value AzureResourceManagerGermany for ContainerRegistryAudience. */
+    public static final ContainerRegistryAudience AZURERESOURCEMANAGERGERMANY = fromString("https://management.microsoftazure.de");
+
+    /** Static value AzureResourceManagerGovernment for ContainerRegistryAudience. */
+    public static final ContainerRegistryAudience AZURERESOURCEMANAGERGOVERNMENT = fromString("https://management.usgovcloudapi.net");
```

Comment:
This name is still being discussed. .Net does not allow _ in the names which honestly will make this much more readable. Also, we need to have consistent names across types.

Reply:
ARMChina etc are already used in another types, hence the user of full AzureResourceManager.

Also, is there any guidance on how we can remove this from intellisense in future? ACR is working on exposing non ARM scopes, so we may want to nudge end customers away from these scopes.

Reply:
Do the property names need to be the same if the serialized value is the same?

Reply:
I have updated the names to follow Java naming conventions. Thanks!